realsense2_camera/src/realsense_node_factory.cpp

```diff
@@ -30,6 +36,8 @@ RealSenseNodeFactory::RealSenseNodeFactory()
 		ros::console::set_logger_level(ROSCONSOLE_DEFAULT_NAME, ros::console::levels::Debug);
 
 	rs2::log_to_console(severity);
+
+	_tf_listener = new tf2_ros::TransformListener(_tf_buffer);
```

Comment:
would suggest using std::unique_ptr instead of handling raw pointers

Reply:
Alright. Will make the changes.

Reply:
Done. Code committed.